Makefile.PL

```diff
@@ -1,15 +1,15 @@
 use strict;
 use warnings;
 require 5.008 ;
+use utf8;
```

Comment:
See the commit message that went with it - `use utf8` prompts the parser to interpret the content encoding properly. Without it, the contributor names were being mojibaked in META.*.